app/graphql/resolvers/auth_base.rb

```diff
@@ -2,7 +2,7 @@
 
 module Resolvers
   class AuthBase < Base
-    def authorized?(**_kwargs)
+    def authorized?(**_args)
```

Comment:
why args?
args - single value arguments
kwargs - key-value arguments

Reply:
as for me `**args` and `**kwars` are both fine)
it's clear that if there are `**` then it's keyword arguments

Reply:
ok